LSTM/TimeSeries-Multivariate/src/LSTMTimeSeriesMultivariate.cpp

```diff
@@ -84,105 +177,127 @@ int main()
   // Max Rho for LSTM 
   const int maxRho = rho;
 
-  // Save/Load model
-  const bool saveModel = true;
-  const bool loadModel = false;
-
   arma::mat dataset;
 
   // In Armadillo rows represent features, columns represent data points.
   cout << "Reading data ..." << endl;
-  data::Load("LSTM/data/Google2016-2019.csv", dataset, true);
-
+  data::Load(dataFile, dataset, true);
 
-  // The dataset CSV file has header, so it's necessary to
-  // get rid of the this row, in Armadillo representation it's the first column
-  // the first col in CSV is not required so removing the first row as well.
+  //The CSV file has a header, so it is necessary to remove it. In Armadillo's 
+  //representation it is the first column.
+  //The first column in the CSV is the date which is not required, therefore 
+  //removing it also (first row in in arma::mat).
   dataset = dataset.submat(1, 1, dataset.n_rows - 1, dataset.n_cols - 1);
 
-  // Scale data for increased numerical stability.
+  // Scale all data into the range (0, 1) for increased numerical stability.
   data::MinMaxScaler scale;
   scale.Fit(dataset);
   scale.Transform(dataset, dataset);
 
-
+  //we have 5 input data columns and 2 output columns (target)
   size_t inputSize = 5, outputSize = 2;
 
+  //We need to represent the input data for RNN in arma::cube (3D matrix)! The 
+  //3rd dimension is the rho number of past data records the RNN uses for 
```

Comment:
```suggestion
  //3rd dimension is rho, the number of past data records the RNN uses for 
```

Let me know if I missed something; this seems a little clearer to me. :+1:

Reply:
OK! This was actually from the original text.

Reply:
No worries, good to fix it anyway.  Thanks for the fix. :+1: